### ISSUE TITLE
Prepare & publish a blog post

### ISSUE THREAD

**Issue by PawelLipski** [VirtusLab/akka-serialization-helper#136]
Basically, README.md but with some changes:
* more fluent narrative
* more graphs/screenshots - maybe even taken from the sphere.it presentation

**Comment by PawelLipski** [comment]
Good first issue for someone to get familiar with the project while already doing some useful work :)

**Comment by miloradvojnovic** [comment]
@PawelLipski should we close this one?

**Comment by PawelLipski** [comment]
Oh! A post to Medium.com like [this one](https://medium.com/virtuslab/take-a-look-at-your-repository-from-a-new-perspective-with-git-machete-plugin-e2e5e4197a0b) would still be useful ;)

**Comment by PawelLipski** [comment]
Okay! so @miloradvojnovic how about you pick up this one in parallel with #148?

**Comment by PawelLipski** [comment]
Marking as `on hold`, as the work is done... but we're blocked on the graphics team now.
